### PR TITLE
docs: Add mnavarroespinosa to core team

### PR DESCRIPTION
**Pull request by herrardo**
Add Manuel Navarro to Core Team

# Pull request template
 

## Motivation and context
Have a better team :)

- [ ] Dependencies. If any, specify:
- [ ] Open issue. If applicable, link:

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [x] Change requires a documentation update

## What is the destination branch of this PR?

- [x] `Main`
- [ ] Other. Specify:

## How has this been tested?

Tests performed according to [testing guidelines](./contributing/tests.md): 

## Checklist:

- [ ] My code follows the **[style guidelines](./CONTRIBUTING.md#style-guides)** of this project.
- [ ] I have performed a **self-review** on my own code.
- [ ] I have **commented** my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the **[documentation](./CONTRIBUTING.md#documentation-style-guide)**.
- [ ] My changes generate **no new warnings**.
- [ ] I have added **tests** that prove my fix is effective or that my feature works.
- [ ] New and existing **unit tests pass locally** with my changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,9 +127,10 @@ the future of search experiences, please check our
...
+|     <a href="https://github.com/herrardo"><img src="https://avatars.githubusercontent.com/u/4663897?v=4" width="100px;" alt=""/><br /><br /><b>Gerardo Vázquez</b></a>     |  <a href="https://github.com/CachedaCodes"><img src="https://avatars.githubusercontent.com/u/7124620?v=4" width="100px;" alt=""/><br /><br /><b>Guillermo Cacheda</b></a>  |     <a href="https://github.com/guillei10"><img src="https://avatars.githubusercontent.com/u/77337158?v=4" width="100px;" alt=""/><br /><br /><b>Guillermo Iglesias</b></a>      |    <a href="https://github.com/mavmaf"><img src="https://avatars.githubusercontent.com/u/77147901?v=4" width="100px;" alt=""/><br /><br /><b>Mavi Fernández</b></a>    | <a href="https://github.com/diegopf"><img src="https://avatars.githubusercontent.com/u/7504736?v=4" width="100px;" alt=""/><br /><br /><b>Diego Pascual</b></a>  |
```
Comment (tiborux) [inline README.md:132]: Remove guille

Comment (herrardo) [reply README.md:132]: oh my

Comment (herrardo) [reply README.md]: Done

Comment (guilleigmu) [reply README.md]: Siempre saludaba

**Review by LuisMartinez15 — APPROVED** [review]
welcome @mnavarroespinosa! sadge for @guilleigmu

**Review by tiborux — APPROVED** [review]

**Pull request merged**